docs/guides/video-tracks.md

```diff
@@ -1,70 +1,83 @@
 # Video Tracks
```

Comment:
It may be good to add that this is an experimental feature as there is no current support in any browser or tool for alternative video tracks.